content/src/main/content/jcr_root/apps/acs-commons/widgets/source/js/multipanel.js

```diff
@@ -72,7 +72,7 @@ ACS.CQ.MultiFieldPanel = CQ.Ext.extend(CQ.Ext.Panel, {
                 return;
             }
 
-            pData[i.key] = i.getValue();
+            pData[i.key] = i.getValue().replace(/"/g, '&quot;');
```

Comment:
Do you know if `i.getValue()` is always guaranteed to be a string (or an obj w a `.replace` method).

Otherwise looks good! Thanks!